start.go

```diff
@@ -129,6 +129,8 @@ func startContainer(context *cli.Context, spec *specs.LinuxSpec) (int, error) {
 	if err != nil {
 		return -1, err
 	}
+	handler := newSignalHandler(tty)
```

Comment:
You cannot move this here because it breaks detach.  Just call `tty.Close()` before returning the error from start